javaparser-core/src/main/java/com/github/javaparser/ast/type/UnionType.java

```diff
@@ -32,6 +30,7 @@ public UnionType(NodeList<ReferenceType<?>> elements) {
     }
 
     public UnionType setElements(NodeList<ReferenceType<?>> elements) {
+        notifyPropertyChange("elements", this.elements, elements);
```

Comment:
Do you think that maybe we should have an enum for the property names? That way we don't get typo's.